### DISCUSSION TITLE
markdown strikethrough syntax `~~ ~~` renders wrong

### DISCUSSION THREAD

**Discussion by yeshiqing** [community/community#39643] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The original sentence is here:
> This is the remain part of the sentence, this is the deleted part of the sentence.

When it comes to GitHub Flavored Markdown Strikethrough syntax:

This is the remain part of the sentence, ~~this is the deleted part of the sentence.~~
This is the remain part of the sentence~~, this is the deleted part of the sentence.~~

The first renders right, but the second has no effect. I just put the `~~` in front of the comma.

Is this a bug or just there must be not an alphanumeric in front of the `~~`?

**Comment by ossanna16** [comment]
Hi there @yeshiqing and welcome to our community. Thank you for reporting this bug!
